services/workflows-service/src/customer/types.ts

```diff
@@ -1,7 +1,35 @@
+import { Customer } from '@prisma/client';
+
 export type TAuthenticationConfiguration = {
   apiType: 'API_KEY' | 'OAUTH2' | 'BASIC_AUTH';
   authValue: string;
   validUntil?: string;
   isValid: boolean;
   webhookSharedSecret: string;
 };
+
+export const CUSTOMER_FEATURES = {
+  ONGOING_AUDIT_REPORT: {
+    name: 'ONGOING_AUDIT_REPORT',
+    definitionName: 'ongoing_audit_report',
+  },
+} as const;
+
+export type TCustomerFeatures = {
+  name: keyof typeof CUSTOMER_FEATURES;
```

Comment:
Features should be pretty agnostic, the features flags should say id a feature is on/off and not provide additional configuration such as definitionName